lightning_pose/models/heatmap_tracker.py

```diff
@@ -81,7 +81,7 @@ def __init__(
         self.upsampling_layers = self.make_upsampling_layers()
         self.initialize_upsampling_layers()
         self.output_shape = output_shape
-        self.temperature = torch.tensor(100, device=self.device)  # soft argmax temp
+        self.temperature = torch.tensor(1000., device=self.device)  # soft argmax temp
```

Comment:
needs to be tested -- varies as the number of pixels in a heatmap
add a comment saying to be careful here.